src/lib/seam/components/SupportedDeviceTable/FilterCategoryMenu.tsx

```diff
@@ -49,20 +49,34 @@ export function FilterCategoryMenu({
           </button>
         )}
       >
-        {usableOptions.map((option, index) => (
-          <MenuItem
-            key={`${index}:${option}`}
-            onClick={() => {
-              if (option === allLabel) {
-                onAllOptionSelect?.()
-              } else {
-                onSelect(option)
+        <div className='seam-supported-device-table-filter-menu-content'>
+          {usableOptions
+            .sort((a, b) => {
```

Comment:
This needs to be moved above like 

```ts
const usableOptions = hideAllOption ? options : [allLabel, ...options].sort(...)
```

otherwise it will mutate the `options` array that is passed in.

Reply:
Thanks! I had forgotten `sort` mutated 😱 .

Updated in https://github.com/seamapi/react/pull/503/commits/912c42dcbc19ec05db4c22933d859fd5e924d864. Also allows for a much shorter `sort` fn this way.